### DISCUSSION TITLE
Page speed insights scores

### DISCUSSION THREAD

**Discussion by devselegento** [scandipwa/scandipwa#2409] [Q&A]
**Description**:
When we are running generate Lighthouse report with Chrome DevTools or we analyze the tech-demo.scandipwa.com with [PageSpeed Insights]( https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Ftech-demo.scandipwa.com%2F) we are getting lower scores (e.g. Performance 62) compared to the ones appearing in scandipwa website.
Is there anything we can do to improve that?

**Comment by aleksandrakorolova** [comment]
Hello
After checking performance for demo website, result is more than 90
![image](https://user-images.githubusercontent.com/73526235/112959465-1f15da80-914c-11eb-8415-48f5ce2b36c7.png)

**Comment by uandiweb** [comment]
Testing the home page speed is not the best practice. For an e-commerce website you should focus on product pages. In many case this is where the customer starts the journey (the "landing page"). 

When I test the product page below I do get a high score, but Google shows it loaded a page that is broken. No wonder it loads fast. I didn't see yet a ScandiPWA product page that fully loads and gets a score above 50 on mobile.

Test this page: https://tech-demo.scandipwa.com/product-68580.html

![broken_page](https://user-images.githubusercontent.com/20729592/113310173-f1f53380-92d5-11eb-90ee-ad96b25ad8ad.jpg)
